Library/Formula/jboss-forge.rb

```diff
@@ -3,9 +3,9 @@
 class JbossForge < Formula
   desc "Tools to help set up and configure a project"
   homepage 'http://forge.jboss.org/'
-  url 'https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=releases&g=org.jboss.forge&a=forge-distribution&v=2.16.2.Final&e=zip&c=offline'
-  version '2.16.2.Final'
-  sha1 'b8588f7b7bb26a0349e33c61cbb1663b6b96040c'
+  url 'https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=releases&g=org.jboss.forge&a=forge-distribution&v=2.17.0.Final&e=zip&c=offline'
+  version '2.17.0.Final'
+  sha1 '38687342e29d0d39b36856dc5aac74b033f72ce3'
```

Comment:
Is this a new requirement? I think I missed the announcement. Will do it. Thanks